packages/plugin-compat/extra/typescript/patch-f347bb4623a8370ac25e4dda78c964e9.diff

```diff
@@ -0,0 +1,2335 @@
+warning: CRLF will be replaced by LF in /lib/lib.d.ts.
```

Comment:
We should probably silence / replace these, unsure if the code `patch-commit` uses also produces these :thinking:

Reply:
It was a mistake, stderr wasn't meant to be piped into the file if the exit code is 0.